qrexec/tests/qrexec_policy_daemon.py

```diff
@@ -181,3 +218,208 @@ async def test_wrong_arg(self, mock_request, async_server, tmp_path):
         await self.send_data(async_server, tmp_path, data)
 
         mock_request.assert_not_called()
+
+    @pytest.mark.asyncio
+    async def test_simple_qrexec_request(
+            self, mock_request, qrexec_server, tmp_path, mock_system):
+
+        for i in server_types:
+            data = b'policy.Eval%s+d c keyword @adminvm\0a\0b' % i
+
+            assert await self.send_data(qrexec_server[i], tmp_path, data, i) == b'result=deny\n'
```

Comment:
Add also some test with "allow" result (by adjusting `mock_request`). Can be this one in fact, as you test very similar thing with deny in `test_not_guivm` test for `policy.EvalSimple` case.